logstash-core/spec/logstash/pipeline_dlq_commit_spec.rb

```diff
@@ -91,7 +91,7 @@ def close() end
     let(:pipeline_id) { "test-dlq" }
 
     it "retrieves proper pipeline-level DLQ writer" do
-      expect_any_instance_of(org.logstash.common.io.DeadLetterQueueWriter).to receive(:close).and_call_original
+      expect_any_instance_of(org.logstash.common.io.DeadLetterQueueWriter).to receive(:close).at_least(:once).and_call_original
```

Comment:
With this change means that JavaPipeline invokes more than 1 times the `close` method on DLQ while the old Pipeline invoked exactly one time, do you  know the motivation of the double call?

Reply:
The first `close` is from `java_pipeline.run`, while the second one is from `java_pipeline.start`
java_pipeline has a different implementation, therefore you see two triggers